### ISSUE TITLE
Activity bar does not support `contrastActiveBorder`

### ISSUE THREAD

**Issue by rzhao271** [microsoft/vscode#167005]
1. Switch to the high contrast dark theme
2. Mouse over the icons of the activity bar
3. :bug: Some icons, like the Accounts icon, show a dashed orange outline. Others, like the SCM view icon, don't.

![Screencap of the steps above](https://user-images.githubusercontent.com/7199958/203411585-7ba003bf-8306-4598-a062-2d6203bcc8ac.gif)

**Comment by sandy081** [comment]
@sbatten I see you have done some changes here before - https://github.com/microsoft/vscode/issues/117584

**Comment by bpasero** [comment]
It looks like the activity bar is still using some old style where things are missing:
- the active view is not indicated with a solid color (see how it works in editor, panel and secondary sidebar)
- on hover there is no dotted border feedback to indicate click area

![image](https://github.com/microsoft/vscode/assets/900690/9ee7d9ba-abc1-4ae3-bfc1-0beb405529cd)

I think whats also needed is to change how the drag-and-drop indicator is working which seems to be on the label in panel and secondary bar but on the label container in the activity bar...

**Comment by VSCodeTriageBot** [comment]
This bug has been fixed in the latest release of [VS Code Insiders](https://code.visualstudio.com/insiders/)!

@rzhao271, you can help us out by commenting `/verified` if things are now working as expected.

If things still don't seem right, please ensure you're on version 00bc1553bafe83896a96e1b2572024083d14515a of Insiders (today's or later - you can use `Help: About` in the command palette to check), and leave a comment letting us know what isn't working as expected.

Happy Coding!
